avidemux_fr.ts

```diff
@@ -112,7 +112,7 @@ Ignorer les informations de timing ?</translation>
     <message>
         <location filename="../ADM_userInterfaces/ADM_dialog/calculator.ui" line="+129"/>
         <source>Custom</source>
-        <translation>Manuel</translation>
+        <translation>Personnalisé</translation>
```

Comment:
Of course, it is not clear from the translation file alone, but the calculator is not a part of the build since at least 14 years. It does no harm to have the strings translated, but the translations will have no effect.